src/WebJobs.Script/Diagnostics/FileTraceWriter.cs

```diff
@@ -21,7 +21,6 @@ public class FileTraceWriter : TraceWriter, IDisposable
         internal const int LogFlushIntervalMs = 1000;
         private readonly string _logFilePath;
         private readonly string _instanceId;
-        private readonly int _processId;
```

Comment:
Process ID isn't needed in the log file name, so removing this. In this changeset, I'm adding Process ID to "Host started" logs, which is something we wanted to do anyways. That should suffice.